src/main/service/publicationService.ts

```diff
@@ -252,4 +254,16 @@ export class PublicationService {
     }
     return publishedTime;
   }
+
+  private static convertPartyRole(nonConvertedPartyRole: string): string {
```

Comment:
Need some unit tests for this bit :)

Reply:
👍 Have added a test to cover a list that has values to be mapped (sonar is now showing 100% coverage for this PR). 
I've also refactored the function again to only use one if as realised it was unnecessary for there to be two separate ones.

Reply:
Really nice :)